extensions/azurePublish/src/node/index.ts

```diff
@@ -150,6 +150,95 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
     /* These methods implement the publish actions */
     /*******************************************************************************************************************************/
     /**
+     * Prepare a bot to be built and deployed by copying the runtime and declarative assets into a temporary folder
+     * @param project
+     * @param settings
+     * @param srcTemplate
+     * @param resourcekey
+     */
+    private init = async (project: any, srcTemplate: string, resourcekey: string, runtime: any) => {
+      try {
+        // point to the declarative assets (possibly in remote storage)
+        const botFiles = project.getProject().files;
+
+        const mode = this.getRuntimeTemplateMode(runtime?.key);
+
+        // include both pre-release and release identifiers here
+        // TODO: eventually we can clean this up when the "old" runtime is deprecated
+        // (old runtime support is the else block below)
+        if (isUsingAdaptiveRuntime(runtime)) {
+          const buildFolder = this.getProjectFolder(resourcekey, mode);
+
+          // clean up from any previous deploys
+          await this.cleanup(resourcekey);
+
+          // copy bot and runtime into projFolder
+          await copy(srcTemplate, buildFolder);
+
+          let manifestPath;
+          for (const file of botFiles) {
+            const pattern = /manifests\/[0-9A-z-]*.json/;
+            if (file.relativePath.match(pattern)) {
```

Comment:
If there are multiple manifest files we automatically just pick the last file?  @luhan2017